nautobot/ipam/views.py

```diff
@@ -361,11 +252,6 @@ class PrefixView(generic.ObjectView):
     )
```

Comment:
Probably should add `"rir"` to the `select_related`? Same on `PrefixListView._queryset` above.